src/GameDataReader/Common/Refractor/V1/Files/UtilityMethodsRefractorV1ConfigFile.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace GameDataReader.Common.Refractor.V1.Files;
+
+internal class UtilityMethodsRefractorV1ConfigFile
+{
+    public string GetProcessPath(string modName)
+    {
+        var matchingProcesses = Process.GetProcessesByName(modName);
+        return matchingProcesses.Length == 1 ? matchingProcesses[0].GetMainModuleFileName(modName) : string.Empty;
+    }
+
+    public string GetConFilePath(string conFile, string gameProcessPath, string gameName)
+    {
+        var appData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+        var executingPath = File.Exists($@"{gameProcessPath}{conFile}") ? $@"{gameProcessPath}{conFile}" : string.Empty;
+        var fallbackPath = File.Exists($@"{appData}\VirtualStore\Program Files (x86)\EA GAMES\{gameName}{conFile}") ? $@"{appData}\VirtualStore\Program Files (x86)\EA GAMES\{gameName}{conFile}" : string.Empty;
+        return File.Exists($@"{gameProcessPath}{conFile}") ? executingPath : fallbackPath;
+    }
+}
```

Comment:
Entire class could be static, unless I am missing something. Would eliminate the need to instantiate any instances of the class.

Reply:
Keep it non-static please, like the other config files.

It's preferred to avoid static classes, unless the class is responsible for only holding data, or doing minor stuff, which won't get extended, tested or replaced (as a dependency to something else)

Reason being, if someone wants to test the logic around it, you cannot inject outside dependencies (through constructor) or replace the class as a dependency itself to another unit of code, with a mocked implementation of it (cannot make an interface on static class)

Reply:
This class isn't a config file, it's just holding some util methods. Maybe the name is a bit misleading?

Reply:
I missed that, sorry, you're right

Then let's rename the file to `RefractorV1PathResolver` or something like that